tests/installation/bootloader_s390.pm

```diff
@@ -277,7 +277,7 @@ sub run {
 
     # We have textmode installation via ssh and the default vnc installation so far
     if (check_var('VIDEOMODE', 'text') || check_var('VIDEOMODE', 'ssh-x')) {
-        type_string("yast.ssh\n");
+        type_string("TERM=linux yast.ssh\n") && record_soft_failure('bsc#1054448');
```

Comment:
why not just call the record_soft_failure in the next line? I don't even understand where type_string returns a true value but apparently it works…